aldryn_apphooks_config/managers/base.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+from django.db.models import ForeignKey, Manager, QuerySet
+
+
+def get_apphook_field_names(model):
+    """
+    Return all foreign key field names for a AppHookConfig based model
+    """
+    from ..models import AppHookConfig  # avoid circular dependencies
+    fields = []
+    for field in model._meta.fields:
+        if (isinstance(field, ForeignKey)
+                and issubclass(field.rel.to, AppHookConfig)):
+            fields.append(field)
+    return [field.name for field in fields]
+
+
+class QuerySetMixin(object):
+
+    def namespace(self, namespace, to=None):
+        """
+        Filter by namespace. Try to guess which field to use in lookup.
+        Accept 'to' argument if you need to specify.
+        """
+        fields = get_apphook_field_names(self.model)
+        if not fields:
+            raise ValueError(
+                "Can't find any relation to an ApphookConfig model "
+                "in {}".format(self.model.__name__)
+            )
+        if to and to not in fields:
+            raise ValueError(
+                "Can't find relation to ApphookConfig model named "
+                "'{}' in '{}'".format(to, self.model.__name__)
```

Comment:
as mentioned in chat, not sure if we will only support 2.7+ so all these ```'{}'.format()``` calls would have to be changed to support 2.6